Mathlib/Data/Real/Basic.lean

```diff
@@ -4,7 +4,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Floris van Doorn
 -/
 import Mathlib.Algebra.Bounds
-import Mathlib.Algebra.Order.Archimedean
 import Mathlib.Algebra.Star.Basic
 import Mathlib.Data.Real.CauSeqCompletion
 
```

Comment:
Could you mention in the file-level docstring what is done in this file, and where one may find the conditionally complete linear order instance? (This is one of the most basic things on reals, so users may be surprised not to find it here).